### PR TITLE
Update Debian/Ubuntu Instruction

### LINKED ISSUE #960: Switch away from apt-key in Debian/Ubuntu instruction
The apt maintainer has recently announced that [apt-key is deprecated](https://blog.jak-linux.org/2021/02/18/apt-2.2/#deprecations) and its use is seen as an [antipattern by other Debian developers](https://michael-prokop.at/blog/2021/02/16/how-to-properly-use-3rd-party-debian-repository-signing-keys-with-apt/).

I would suggest to adopt something like the [Kubernetes instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management) or even follow the guide in the [Debian wiki](https://wiki.debian.org/DebianRepository/UseThirdParty) for packaging the key inside the repository (in a package named `adoptopenjdk-archive-keyring`) to allow automatic key rollovers.

### PR DESCRIPTION
**Pull request by Martins-O-U**
##### Checklist

- [x] `npm test` passes
- [x] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

Fixes #960

### CODE REVIEW

**Review by smlambert — COMMENTED** [review]
First review comments, @Martins-O-U can you update the PR title to something more meaningful like "Update Debian/Ubuntu instruction" and can you update the description to include a "Fixes #960" statement so that when the PR will be merged it will automatically close the github issue?

**Comment by Martins-O-U** [comment]
Hello @smlambert, can this Pull Request be given a second look after the title update?

File: src/handlebars/installation.handlebars [hunk 1]
```diff
@@ -1,55 +1,59 @@
-{{> header title='Installation | ' style1='<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/styles/obsidian.min.css">' }}
+{{> header title='Installation | ' style1='
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/styles/obsidian.min.css">' }}
```
Comment (karianna) [inline src/handlebars/installation.handlebars:2]: should stay on one line

**Comment by karianna** [comment]
@Martins-O-U it's a bit hard to see the change due to all of the whitespace changes.  Did you apply some sort of a formatter?  If it's possible to redo this PR without those changes would be preferable...

**Review by karianna — CHANGES_REQUESTED** [review]
it's a bit hard to see the change due to all of the whitespace changes. Did you apply some sort of a formatter? If it's possible to redo this PR without those changes would be preferable...

**Comment by Martins-O-U** [comment]
Hello @karianna and @smlambert,
I am sorry my code formatter was on when I worked on the task, thus it formatted some of the lines. It has now been fixed.  
On this PR, you would still see the lines where the formatting was removed displaying as effected changes. However, the section with actual edits requiring review is between line 339 - 465 (Debian/Ubuntu and its distributions installation)

Please, would you be able to review based on lines 339 - 465, or will it be better if I close this PR and make an entirely new PR?

**Comment by karianna** [comment]
If you could make a new PR that would help (small whitespace changes we can live with, but some of the changes have wrapped lines etc)

**Pull request closed without merging**